<launch>
   <node name="location_node" pkg="lx_camera_ros" type="location_node" output="screen">
    <!-- 相机IP，默认为空时按设备索引获取 -->
    <param name="ip"                         type="string"      value="192.168.100.82" />
    <!-- 图像显示使能 -->
    <param name="is_show"                    type="bool"        value="true" />
    <!-- 接收或发布ROS话题名 -->
    <param name="LxCamera_UploadScan"        type="string"      value="/sim/scan" />
    <param name="LxCamera_UploadOdom"        type="string"      value="/sim/odom" />
    <param name="LxCamera_UploadLaserPose"   type="string"      value="/sim/scan_pose" />
    <param name="LxCamera_Error"             type="string"      value="LxCamera_Error" />
    <param name="LxCamera_Command"           type="string"      value="LxCamera_Command" />
    <param name="LxCamera_Mapping"           type="string"      value="LxCamera_Mapping" />
    <param name="LxCamera_Location"          type="string"      value="LxCamera_Location" />
    <param name="LxCamera_SetParam"          type="string"      value="LxCamera_SetParam" />
    <param name="LxCamera_SwitchMap"         type="string"      value="LxCamera_SwitchMap" />
    <param name="LxCamera_DownloadMap"       type="string"      value="LxCamera_DownloadMap" />
    <param name="LxCamera_UploadMap"         type="string"      value="LxCamera_UploadMap" />
    <param name="LxCamera_LocationResult"    type="string"      value="LxCamera_LocationResult" />
    <param name="LxCamera_UploadReloc"       type="string"      value="LxCamera_UploadReloc" />
    <!-- 自动曝光期望值，范围[0-100], 默认:50 -->
    <param name="auto_exposure_value"        type="int"         value="50" />
    <!-- 建图使能, true or false -->
    <param name="mapping_mode"               type="bool"        value="true" />
    <!-- 定位使能, true or false -->
    <param name="localization_mode"          type="bool"        value="false" />
    <!-- 重要：定位时地图名需有效，已上传相机并存在；建图时,如果相机中无地图可默认输入"example_map1" -->
    <param name="map_name"          type="string"        value="example_map1" />
    <!-- 相机至小车外参, 单位:米, 度数, 格式[x, y, z, yaw, pitch, roll] -->
    <rosparam param="camera_extrinsic_param"> [0.34, 0.00, 1.3, -90, 0, 0] </rosparam>
    <!-- 激光雷达至小车外参, 单位:米, 度数, 格式[x, y, yaw] -->
    <rosparam param="laser_extrinsic_param"> [0.34, 0.11, 0.0] </rosparam>
   </node>

</launch>
